entry_types/scrolled/lib/pageflow_scrolled/seeds.rb

```diff
@@ -58,18 +68,27 @@ def say_creating_scrolled_entry(entry)
       say("   sample scrolled entry '#{entry.title}'\n")
     end
 
-    def create_image_files(draft_entry, image_file_data_by_name)
-      image_file_data_by_name.transform_values do |data|
-        say("     creating image file from #{data['url']}")
+    def create_files(draft_entry, file_type, file_data_by_name)
+      file_data_by_name.transform_values do |data|
+        say("     creating #{file_type} file from #{data['url']}")
+
+        file = draft_entry.create_file!(Pageflow::BuiltInFileType.send(file_type),
+                                        state: file_type == :image ? 'processed' : 'uploading',
+                                        attachment: URI.parse(data['url']),
+                                        configuration: data['configuration'])
+        if file_type == :video
+          if draft_entry.entry_title == 'Storybook seed'
```

Comment:
If this was an option like @skip_encoding@, it could be set by the rake task without needing to hard code a name. It would also be easier to test then.

Reply:
Option added. Refactored attributes to keyword argument and added options as separate argument